src/chain_spec.rs

```diff
@@ -94,49 +114,30 @@ impl Alternative {
     }
 }
 
-const NICOLE: [u8; 32] = [
-    68, 169, 150, 190, 177, 238, 247, 189, 202, 185, 118, 171, 109, 44, 162, 97, 4, 131, 65, 100,
-    236, 242, 143, 179, 117, 96, 5, 118, 252, 198, 235, 15,
-];
-
-fn testnet_genesis(
-    initial_authorities: Vec<AuthorityId>,
-    endowed_accounts: Vec<AccountId>,
+fn testnet_genesis(initial_authorities: Vec<(AuraId, GrandpaId)>,
     root_key: AccountId,
-) -> GenesisConfig {
+    endowed_accounts: Vec<AccountId>,
+    _enable_println: bool) -> GenesisConfig {
     GenesisConfig {
-		consensus: Some(ConsensusConfig {
-			code: include_bytes!("../runtime/wasm/target/wasm32-unknown-unknown/release/utxo_runtime_wasm.compact.wasm").to_vec(),
-			authorities: initial_authorities.clone(),
-		}),
-		system: None,
-		timestamp: Some(TimestampConfig {
-			minimum_period: 5, // 10 second block time.
-		}),
-		indices: Some(IndicesConfig {
-			ids: endowed_accounts.clone(),
-		}),
-		balances: Some(BalancesConfig {
-			transaction_base_fee: 1,
-			transaction_byte_fee: 0,
-			existential_deposit: 500,
-			transfer_fee: 0,
-			creation_fee: 0,
-			balances: endowed_accounts.iter().cloned().map(|k|(k, 1 << 60)).collect(),
-			vesting: vec![],
-		}),
-		sudo: Some(SudoConfig {
-			key: root_key,
-		}),
-        utxo: Some(UtxoConfig {
-            initial_utxo: vec![
-				utxo::TransactionOutput {
-					value: utxo::Value::max_value(),
-					pubkey: H256::from_slice(&NICOLE),
-					salt: 0,
-				}
-			],
-            ..Default::default()
-        }),
-	}
+      system: Some(SystemConfig {
+        code: WASM_BINARY.to_vec(),
+        changes_trie_config: Default::default(),
+      }),
+      indices: Some(IndicesConfig {
+        ids: endowed_accounts.clone(),
+      }),
+      balances: Some(BalancesConfig {
+        balances: endowed_accounts.iter().cloned().map(|k|(k, 1 << 60)).collect(),
+        vesting: vec![],
+      }),
+      sudo: Some(SudoConfig {
+        key: root_key,
+      }),
+      aura: Some(AuraConfig {
+        authorities: initial_authorities.iter().map(|x| (x.0.clone())).collect(),
+      }),
+      grandpa: Some(GrandpaConfig {
+        authorities: initial_authorities.iter().map(|x| (x.1.clone(), 1)).collect(),
+      }),
+    }
```

Comment:
Issue: your new chain_spec breaks the testnet/ui demo instructions in README section #demo-polkadot-UI. 

I agree with this refactoring (thank you) - it makes the repo itself much cleaner.

However, Alice/other accounts are no longer preloaded with any utxo at genesis, which the UI demo assumes.

Suggestion: also update the demo readme to make this tutorial work with the new workshop set up. i.e. just make the user sudo -> mint into Alice, etc. Note that then, all other steps and the encoded `hints` must also be updated.